fabric-common/lib/BlockDecoder.js

```diff
@@ -698,53 +766,53 @@ function decodeEndorserTransaction(trans_bytes) {
-	logger.debug('decodeTokenTransaction start');
-
-	const data = {};
-	try {
-		const transaction = fabprotos.token.TokenTransaction.decode(trans_bytes);
-
-		if (transaction && transaction.token_action) {
-			const token_action = transaction.token_action;
-			const action_data = token_action[token_action.data];
-
-			// decode output tokens in action_data
-			const tokens = [];
-			for (const proto_token of action_data.outputs) {
-				const token = {};
-				if (proto_token.owner) {
-					// owner can be null for redeem
-					token.owner = {type: proto_token.owner.type, raw: proto_token.owner.raw.toBuffer()};
-				}
-				token.type = proto_token.type;
-				token.quantity = proto_token.quantity;
-				tokens.push(token);
-			}
-
-			// decode input token ids in action_data
-			const tokenIds = [];
-			if (action_data.inputs) {
-				for (const proto_input of action_data.inputs) {
-					const tokenId = {index: proto_input.index, tx_id: proto_input.tx_id};
-					tokenIds.push(tokenId);
-				}
-			}
-
-			// construct decoded data to return
-			data.token_action = {data: token_action.data};
-			if (tokenIds.length > 0) {
-				data.token_action[token_action.data] = {inputs: tokenIds, outputs: tokens};
-			} else {
-				data.token_action[token_action.data] = {outputs: tokens};
-			}
-		}
-	} catch (error) {
-		logger.error(' Unable to decodeTokenTransaction :: %s', error);
-	}
-
-	return data;
-}
+// function decodeTokenTransaction(trans_bytes) {
+// 	logger.debug('decodeTokenTransaction start');
+
+// 	const data = {};
+// 	try {
+// 		const transaction = fabprotos.token.TokenTransaction.decode(trans_bytes);
+
+// 		if (transaction && transaction.token_action) {
+// 			const token_action = transaction.token_action;
+// 			const action_data = token_action[token_action.data];
+
+// 			// decode output tokens in action_data
+// 			const tokens = [];
+// 			for (const proto_token of action_data.outputs) {
+// 				const token = {};
+// 				if (proto_token.owner) {
+// 					// owner can be null for redeem
+// 					token.owner = {type: proto_token.owner.type, raw: proto_token.owner.raw.toBuffer()};
+// 				}
+// 				token.type = proto_token.type;
+// 				token.quantity = proto_token.quantity;
+// 				tokens.push(token);
+// 			}
+
+// 			// decode input token ids in action_data
+// 			const tokenIds = [];
+// 			if (action_data.inputs) {
+// 				for (const proto_input of action_data.inputs) {
+// 					const tokenId = {index: proto_input.index, tx_id: proto_input.tx_id};
+// 					tokenIds.push(tokenId);
+// 				}
+// 			}
+
+// 			// construct decoded data to return
+// 			data.token_action = {data: token_action.data};
+// 			if (tokenIds.length > 0) {
+// 				data.token_action[token_action.data] = {inputs: tokenIds, outputs: tokens};
+// 			} else {
+// 				data.token_action[token_action.data] = {outputs: tokens};
+// 			}
+// 		}
+// 	} catch (error) {
+// 		logger.error(' Unable to decodeTokenTransaction :: %s', error);
+// 	}
+
+// 	return data;
+// }
```

Comment:
is this function no longer needed?  if not, we should delete it

Reply:
I left it as I heard rumblings of token coming back, but your right I should just remove it